.devcontainer/post-create.sh

```diff
@@ -4,10 +4,11 @@ chmod +x /workspace/comfystream/docker/entrypoint.sh
 cd /workspace/comfystream
 
 # Install Comfystream in editable mode.
-/workspace/comfystream/docker/entrypoint.sh --editable
+echo -e "\e[32mInstalling Comfystream in editable mode...\e[0m"
```

Comment:
I removed the usage of the entrypoint for such a simple command to ensure we don't have nested shells.

Reply:
I really like the log message, it's informative